osu.Game/Rulesets/Mods/ModHidden.cs

```diff
@@ -49,36 +40,10 @@ public ScoreRank AdjustRank(ScoreRank rank, double accuracy)
 
         protected override void ApplyIncreasedVisibilityState(DrawableHitObject hitObject, ArmedState state)
         {
-#pragma warning disable 618
-            ApplyFirstObjectIncreaseVisibilityState(hitObject, state);
-#pragma warning restore 618
         }
 
         protected override void ApplyNormalVisibilityState(DrawableHitObject hitObject, ArmedState state)
         {
-#pragma warning disable 618
-            ApplyHiddenState(hitObject, state);
-#pragma warning restore 618
-        }
```

Comment:
Empty overrides now, can be removed.

Reply:
This caused some missing inheritance, not sure if I should duplicate these missing overrides or revert this

Reply:
Since `ApplyIncreasedVisibilityState` does nothing on most hidden mods, the override for it can return back here.

Reply:
Never mind, there aren't many usages here, just override on each implementation instead.

Reply:
Damn I just pushed the change